### ISSUE TITLE
Compiler crash with PackedIntArray

### ISSUE THREAD

**Issue by ProkopRandacek** [ziglang/zig#14596]
### Zig Version

0.11.0-dev.1416+8484df5bd

### Steps to Reproduce and Observed Behavior

Hi! :D

I managed to reduce the error into this sample:
```zig
const std = @import("std");

pub const PieceType = enum(u3) { Queen, Rook, Bishop, Knight, King, Pawn, None };

pub const Position = struct {
    pcr: std.PackedIntArray(u3, 64) = std.PackedIntArray(u3, 64).initAllTo(@enumToInt(PieceType.None)),
};

pub fn main() !void {
    _ = Position{};
}
```

and this is the compiler log:
```
~/code/uz 1 % ~/source/zig/stage3/bin/zig build
thread 6735 panic: index out of bounds: index 2, len 1
Analyzing /home/prokop/source/zig/stage3/lib/zig/std/packed_int_array.zig: packed_int_array.zig:PackedIntIo(u3,.Little).setBits
      %504 = dbg_block_begin()
      %505 = dbg_stmt(2, 13)
      %506 = bit_size_of(%497) 
      %507 = dbg_var_val(%506, "container_bits")
      %508 = dbg_stmt(3, 13)
      %510 = decl_ref("std") 
      %511 = dbg_stmt(3, 30)
      %512 = field_ptr(%510, "math") 
      %513 = dbg_stmt(3, 35)
      %514 = field_call_bind(%512, "Log2Int") 
      %515 = dbg_stmt(3, 43)
      %516 = call(.auto, %514, [
        {
          %517 = break_inline(%516, %497)
        },
      ]) 
      %519 = save_err_ret_index(%516)
      %520 = dbg_var_val(%516, "Shift")
      %521 = dbg_stmt(5, 13)
      %522 = int(8)
      %523 = dbg_stmt(5, 42)
      %524 = div(%499, %522) 
      %525 = dbg_var_val(%524, "start_byte")
      %526 = dbg_stmt(6, 13)
      %527 = int(8)
      %528 = dbg_stmt(6, 60)
      %529 = mul(%524, %527) 
      %530 = dbg_stmt(6, 46)
      %531 = sub(%499, %529) 
      %532 = dbg_var_val(%531, "head_keep_bits")
      %533 = dbg_stmt(7, 13)
      %534 = closure_get(%150) 
      %535 = dbg_stmt(7, 63)
      %536 = add(%534, %531) 
      %537 = dbg_stmt(7, 51)
      %538 = sub(%506, %536) 
      %539 = dbg_var_val(%538, "tail_keep_bits")
      %540 = dbg_stmt(8, 13)
      %542 = closure_get(%295) 
      %543 = switch_cond(%542) 
      %544 = dbg_stmt(8, 40)
      %545 = typeof(%543) 
      %547 = enum_literal("Big") 
      %548 = as_node(%545, %547) 
      %555 = enum_literal("Little") 
      %556 = as_node(%545, %555) 
      %546 = switch_block(%543,
        %548 => {
          %549 = dbg_block_begin()
          %550 = dbg_stmt(9, 25)
          %551 = int_cast(%516, %538) 
          %552 = store_to_block_ptr(%541, %551)
          %554 = break(%546, %551)
        },
        %556 => {
          %557 = dbg_block_begin()
          %558 = dbg_stmt(10, 28)
          %559 = int_cast(%516, %531) 
          %560 = store_to_block_ptr(%541, %559)
          %562 = break(%546, %559)
        }) 
      %564 = save_err_ret_index(%546)
      %565 = dbg_var_val(%546, "keep_shift")
      %566 = dbg_stmt(14, 27)
      %567 = closure_get(%390) 
      %568 = bitcast(%567, %502) 
      %569 = int_cast(%497, %568) 
      %570 = typeof_log2_int_type(%569) 
      %571 = as_shift_operand(%570, %546) 
      %572 = dbg_stmt(14, 69)
      %573 = shl(%569, %571) 
      %574 = dbg_var_val(%573, "value")
      %575 = dbg_stmt(17, 32)
      %576 = ptr_type(%497, One, align(@Zir.Inst.Ref.one)) 
      %577 = ref(%495) 
      %578 = as_node(@Zir.Inst.Ref.usize_type, %524) 
      %579 = dbg_stmt(17, 68)
      %580 = elem_ptr_node(%577, %578) 
      %581 = ptr_cast(%576, %580) 
      %582 = dbg_var_val(%581, "target_ptr")
      %583 = dbg_stmt(18, 13)
      %584 = alloc_inferred_mut() 
      %585 = validate_deref(%581) 
      %586 = load(%581) 
      %587 = store_to_inferred_ptr(%584, %586)
      %588 = resolve_inferred_alloc(%584) 
      %589 = dbg_var_ptr(%584, "target")
      %590 = dbg_stmt(20, 17)
      %596 = block({
        %591 = closure_get(%295) 
        %592 = decl_val("native_endian") 
        %593 = cmp_neq(%591, %592) 
        %594 = as_node(@Zir.Inst.Ref.bool_type, %593) 
        %595 = condbr(%594, {
          %597 = dbg_block_begin()
          %598 = dbg_stmt(20, 42)
          %599 = load(%584) 
          %600 = byte_swap(%599) 
          %601 = store_node(%584, %600) 
          %602 = dbg_block_end()
          %603 = break(%596, @Zir.Inst.Ref.void_value)
        }, {
          %604 = break(%596, @Zir.Inst.Ref.void_value)
        }) 
      }) 
      %605 = ensure_result_used(%596) 
      %606 = dbg_stmt(23, 30)
      %607 = decl_ref("std") 
      %608 = dbg_stmt(23, 53)
      %609 = field_ptr(%607, "math") 
      %610 = dbg_stmt(23, 58)
      %611 = field_call_bind(%609, "maxInt") 
      %612 = dbg_stmt(23, 65)
      %613 = call(.auto, %611, [
        {
          %614 = closure_get(%390) 
          %615 = break_inline(%613, %614)
        },
      ]) 
      %616 = int_cast(%497, %613) 
      %617 = typeof_log2_int_type(%616) 
      %618 = as_shift_operand(%617, %546) 
      %619 = dbg_stmt(23, 74)
      %620 = shl(%616, %618) 
      %621 = dbg_var_val(%620, "inv_mask")
      %622 = dbg_stmt(24, 13)
      %623 = bit_not(%620) 
      %624 = dbg_var_val(%623, "mask")
      %625 = dbg_stmt(25, 13)
      %626 = load(%584) 
      %627 = typeof(%626) 
      %628 = bit_and(%626, %623) 
      %629 = store(%584, %628)
      %630 = dbg_stmt(28, 13)
      %631 = load(%584) 
      %632 = typeof(%631) 
      %633 = bit_or(%631, %573) 
      %634 = store(%584, %633)
      %635 = dbg_stmt(30, 17)
      %641 = block({
        %636 = closure_get(%295) 
        %637 = decl_val("native_endian") 
        %638 = cmp_neq(%636, %637) 
        %639 = as_node(@Zir.Inst.Ref.bool_type, %638) 
        %640 = condbr(%639, {
          %642 = dbg_block_begin()
          %643 = dbg_stmt(30, 42)
          %644 = load(%584) 
          %645 = byte_swap(%644) 
          %646 = store_node(%584, %645) 
          %647 = dbg_block_end()
          %648 = break(%641, @Zir.Inst.Ref.void_value)
        }, {
          %649 = break(%641, @Zir.Inst.Ref.void_value)
        }) 
      }) 
      %650 = ensure_result_used(%641) 
      %651 = dbg_stmt(33, 13)
      %652 = validate_deref(%581) 
      %653 = load(%584) 
    > %654 = store_node(%581, %653) 
      %655 = dbg_block_end()
      %656 = restore_err_ret_index(%503, @Zir.Inst.Ref.none)
      %657 = break(%503, @Zir.Inst.Ref.void_value)
    For full context, use the command
      zig ast-check -t /home/prokop/source/zig/stage3/lib/zig/std/packed_int_array.zig

  in /home/prokop/source/zig/stage3/lib/zig/std/packed_int_array.zig: packed_int_array.zig:PackedIntIo(u3,.Little).setBits
    > %503 = block({%504..%657}) 
  in /home/prokop/source/zig/stage3/lib/zig/std/packed_int_array.zig: packed_int_array.zig:PackedIntIo(u3,.Little).set
    > %479 = call(nodiscard .auto, %477, [
        {%480},
        {%481, %482},
        {%483},
        {%484},
      ]) 
  in /home/prokop/source/zig/stage3/lib/zig/std/packed_int_array.zig: packed_int_array.zig:PackedIntIo(u3,.Little).set
    > %418 = block({%419..%487}) 
  in /home/prokop/source/zig/stage3/lib/zig/std/packed_int_array.zig: packed_int_array.zig:PackedIntArrayEndian(u3,.Little,64).set
    > %1160 = call(.auto, %1158, [
        {%1161..%1164},
        {%1165},
        {%1166},
        {%1167},
      ]) 
  in /home/prokop/source/zig/stage3/lib/zig/std/packed_int_array.zig: packed_int_array.zig:PackedIntArrayEndian(u3,.Little,64).set
    > %1142 = block({%1143..%1175}) 
  in /home/prokop/source/zig/stage3/lib/zig/std/packed_int_array.zig: packed_int_array.zig:PackedIntArrayEndian(u3,.Little,64).setAll
    > %1220 = call(nodiscard .auto, %1218, [
        {%1221, %1222},
        {%1223},
      ]) 
  in /home/prokop/source/zig/stage3/lib/zig/std/packed_int_array.zig: packed_int_array.zig:PackedIntArrayEndian(u3,.Little,64).setAll
    > %1213 = block({%1214..%1226}) 
  in /home/prokop/source/zig/stage3/lib/zig/std/packed_int_array.zig: packed_int_array.zig:PackedIntArrayEndian(u3,.Little,64).setAll
    > %1202 = block({%1212..%1227}) 
  in /home/prokop/source/zig/stage3/lib/zig/std/packed_int_array.zig: packed_int_array.zig:PackedIntArrayEndian(u3,.Little,64).setAll
    > %1200 = condbr(%1199, {%1204..%1228}, {%1229}) 
  in /home/prokop/source/zig/stage3/lib/zig/std/packed_int_array.zig: packed_int_array.zig:PackedIntArrayEndian(u3,.Little,64).setAll
    > %1201 = block({%1195..%1200}) 
  in /home/prokop/source/zig/stage3/lib/zig/std/packed_int_array.zig: packed_int_array.zig:PackedIntArrayEndian(u3,.Little,64).setAll
    > %1194 = loop({%1201, %1203}) 
  in /home/prokop/source/zig/stage3/lib/zig/std/packed_int_array.zig: packed_int_array.zig:PackedIntArrayEndian(u3,.Little,64).setAll
    > %1188 = block({%1189..%1233}) 
  in /home/prokop/source/zig/stage3/lib/zig/std/packed_int_array.zig: packed_int_array.zig:PackedIntArrayEndian(u3,.Little,64).initAllTo
    > %1070 = call(nodiscard .auto, %1068, [
        {%1071},
      ]) 
  in /home/prokop/source/zig/stage3/lib/zig/std/packed_int_array.zig: packed_int_array.zig:PackedIntArrayEndian(u3,.Little,64).initAllTo
    > %1046 = block({%1047..%1081}) 
  in src/main.zig: main.zig:Position
    > %32 = call(.compile_time, %30, [
        {%33..%36},
      ]) node_offset:6:39 to :6:103
  in src/main.zig: main.zig:main
    > %44 = struct_init_empty(%43) node_offset:10:9 to :10:19
  in src/main.zig: main.zig:main
    > %40 = block({%41..%48}) node_offset:9:21 to :9:22
  in /home/prokop/source/zig/stage3/lib/zig/std/start.zig: start.zig:callMain
    > %3080 = is_non_err(%3079) 
  in /home/prokop/source/zig/stage3/lib/zig/std/start.zig: start.zig:callMain
    > %3082 = block({%3075..%3081}) 
  in /home/prokop/source/zig/stage3/lib/zig/std/start.zig: start.zig:callMain
    > %3072 = block({%3073..%3219}) 
  in /home/prokop/source/zig/stage3/lib/zig/std/start.zig: start.zig:callMain
    > %2976 = switch_block(%2973,
        else => {%3222..%3226},
        %2978 => {%2979..%2992},
        %2994 => {%2995..%3014},
        %3017 => {%3015..%3068},
        %3070 => {%3071..%3221}) 
  in /home/prokop/source/zig/stage3/lib/zig/std/start.zig: start.zig:callMain
    > %2960 = block({%2961..%3230}) 
  in /home/prokop/source/zig/stage3/lib/zig/std/start.zig: start.zig:initEventLoopAndCallMain
    > %2676 = builtin_call(%2674, %2675, @Zir.Inst.Ref.empty_struct) 
  in /home/prokop/source/zig/stage3/lib/zig/std/start.zig: start.zig:initEventLoopAndCallMain
    > %2520 = block({%2521..%2684}) 
  in /home/prokop/source/zig/stage3/lib/zig/std/start.zig: start.zig:callMainWithArgs
    > %2309 = call(.auto, %2307, []) 
  in /home/prokop/source/zig/stage3/lib/zig/std/start.zig: start.zig:callMainWithArgs
    > %2278 = block({%2279..%2317}) 
  in /home/prokop/source/zig/stage3/lib/zig/std/start.zig: start.zig:posixCallMainAndExit
    > %2141 = builtin_call(%2138, %2139, %2140) 
  in /home/prokop/source/zig/stage3/lib/zig/std/start.zig: start.zig:posixCallMainAndExit
    > %2137 = call(nodiscard .auto, %2135, [
        {%2138..%2142},
      ]) 
  in /home/prokop/source/zig/stage3/lib/zig/std/start.zig: start.zig:posixCallMainAndExit
    > %1752 = block({%1753..%2145}) 

???:?:?: 0x86e158a in writeToMemory (crt/x86_64/crti.s)
???:?:?: 0x86d96c3 in storePtrVal (crt/x86_64/crti.s)
???:?:?: 0x821c0ca in storePtr2 (crt/x86_64/crti.s)
???:?:?: 0x832339e in zirStoreNode (crt/x86_64/crti.s)
???:?:?: 0x7fb1fd4 in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x7d73515 in analyzeBodyBreak (crt/x86_64/crti.s)
???:?:?: 0x7fb51e1 in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x8216848 in analyzeBody (crt/x86_64/crti.s)
???:?:?: 0x87986ef in analyzeCall (crt/x86_64/crti.s)
???:?:?: 0x826c278 in zirCall (crt/x86_64/crti.s)
???:?:?: 0x7fa3f3a in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x7d73515 in analyzeBodyBreak (crt/x86_64/crti.s)
???:?:?: 0x7fb51e1 in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x8216848 in analyzeBody (crt/x86_64/crti.s)
???:?:?: 0x87986ef in analyzeCall (crt/x86_64/crti.s)
???:?:?: 0x826c278 in zirCall (crt/x86_64/crti.s)
???:?:?: 0x7fa3f3a in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x7d73515 in analyzeBodyBreak (crt/x86_64/crti.s)
???:?:?: 0x7fb51e1 in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x8216848 in analyzeBody (crt/x86_64/crti.s)
???:?:?: 0x87986ef in analyzeCall (crt/x86_64/crti.s)
???:?:?: 0x826c278 in zirCall (crt/x86_64/crti.s)
???:?:?: 0x7fa3f3a in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x7d73515 in analyzeBodyBreak (crt/x86_64/crti.s)
???:?:?: 0x7fb51e1 in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x7d73515 in analyzeBodyBreak (crt/x86_64/crti.s)
???:?:?: 0x7fb51e1 in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x7d73515 in analyzeBodyBreak (crt/x86_64/crti.s)
???:?:?: 0x7fb6d80 in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x7d73515 in analyzeBodyBreak (crt/x86_64/crti.s)
???:?:?: 0x7fb51e1 in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x7d73515 in analyzeBodyBreak (crt/x86_64/crti.s)
???:?:?: 0x7fb4b93 in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x7d73515 in analyzeBodyBreak (crt/x86_64/crti.s)
???:?:?: 0x7fb51e1 in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x8216848 in analyzeBody (crt/x86_64/crti.s)
???:?:?: 0x87986ef in analyzeCall (crt/x86_64/crti.s)
???:?:?: 0x826c278 in zirCall (crt/x86_64/crti.s)
???:?:?: 0x7fa3f3a in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x7d73515 in analyzeBodyBreak (crt/x86_64/crti.s)
???:?:?: 0x7fb51e1 in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x8216848 in analyzeBody (crt/x86_64/crti.s)
???:?:?: 0x87986ef in analyzeCall (crt/x86_64/crti.s)
???:?:?: 0x826c278 in zirCall (crt/x86_64/crti.s)
???:?:?: 0x7fa3f3a in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x7d73515 in analyzeBodyBreak (crt/x86_64/crti.s)
???:?:?: 0x878d2a3 in resolveBody (crt/x86_64/crti.s)
???:?:?: 0x883ecc7 in semaStructFields (crt/x86_64/crti.s)
???:?:?: 0x8348457 in resolveTypeFieldsStruct (crt/x86_64/crti.s)
???:?:?: 0x7fcb8d4 in resolveTypeFields (crt/x86_64/crti.s)
???:?:?: 0x838a1ef in structInitEmpty (crt/x86_64/crti.s)
???:?:?: 0x82b0053 in zirStructInitEmpty (crt/x86_64/crti.s)
???:?:?: 0x7fa8c63 in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x87d3009 in resolveBlockBody (crt/x86_64/crti.s)
???:?:?: 0x8331212 in zirBlock (crt/x86_64/crti.s)
???:?:?: 0x7fb4e29 in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x8216848 in analyzeBody (crt/x86_64/crti.s)
???:?:?: 0x7f863ba in analyzeFnBody (crt/x86_64/crti.s)
???:?:?: 0x7d529d1 in ensureFuncBodyAnalyzed (crt/x86_64/crti.s)
???:?:?: 0x8860077 in ensureFuncBodyAnalyzed (crt/x86_64/crti.s)
???:?:?: 0x83743ca in resolveInferredErrorSet (crt/x86_64/crti.s)
???:?:?: 0x8335951 in analyzeIsNonErrComptimeOnly (crt/x86_64/crti.s)
???:?:?: 0x87afc5c in analyzeIsNonErr (crt/x86_64/crti.s)
???:?:?: 0x827cda7 in zirIsNonErr (crt/x86_64/crti.s)
???:?:?: 0x7fa68cb in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x87d3009 in resolveBlockBody (crt/x86_64/crti.s)
???:?:?: 0x8331212 in zirBlock (crt/x86_64/crti.s)
???:?:?: 0x7fb4e29 in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x87d3009 in resolveBlockBody (crt/x86_64/crti.s)
???:?:?: 0x8331212 in zirBlock (crt/x86_64/crti.s)
???:?:?: 0x7fb4e29 in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x87d3009 in resolveBlockBody (crt/x86_64/crti.s)
???:?:?: 0x828dbb9 in zirSwitchBlock (crt/x86_64/crti.s)
???:?:?: 0x7fa7dd8 in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x87d3009 in resolveBlockBody (crt/x86_64/crti.s)
???:?:?: 0x8331212 in zirBlock (crt/x86_64/crti.s)
???:?:?: 0x7fb4e29 in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x8216848 in analyzeBody (crt/x86_64/crti.s)
???:?:?: 0x87986ef in analyzeCall (crt/x86_64/crti.s)
???:?:?: 0x82cf9ed in zirBuiltinCall (crt/x86_64/crti.s)
???:?:?: 0x7fab39b in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x87d3009 in resolveBlockBody (crt/x86_64/crti.s)
???:?:?: 0x8331212 in zirBlock (crt/x86_64/crti.s)
???:?:?: 0x7fb4e29 in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x8216848 in analyzeBody (crt/x86_64/crti.s)
???:?:?: 0x87986ef in analyzeCall (crt/x86_64/crti.s)
???:?:?: 0x826c278 in zirCall (crt/x86_64/crti.s)
???:?:?: 0x7fa3f3a in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x87d3009 in resolveBlockBody (crt/x86_64/crti.s)
???:?:?: 0x8331212 in zirBlock (crt/x86_64/crti.s)
???:?:?: 0x7fb4e29 in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x8216848 in analyzeBody (crt/x86_64/crti.s)
???:?:?: 0x87986ef in analyzeCall (crt/x86_64/crti.s)
???:?:?: 0x82cf9ed in zirBuiltinCall (crt/x86_64/crti.s)
???:?:?: 0x7fab39b in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x7d73515 in analyzeBodyBreak (crt/x86_64/crti.s)
???:?:?: 0x878d2a3 in resolveBody (crt/x86_64/crti.s)
???:?:?: 0x826bd94 in zirCall (crt/x86_64/crti.s)
???:?:?: 0x7fa3f3a in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x87d3009 in resolveBlockBody (crt/x86_64/crti.s)
???:?:?: 0x8331212 in zirBlock (crt/x86_64/crti.s)
???:?:?: 0x7fb4e29 in analyzeBodyInner (crt/x86_64/crti.s)
???:?:?: 0x8216848 in analyzeBody (crt/x86_64/crti.s)
???:?:?: 0x7f863ba in analyzeFnBody (crt/x86_64/crti.s)
???:?:?: 0x7d529d1 in ensureFuncBodyAnalyzed (crt/x86_64/crti.s)
???:?:?: 0x7d50515 in processOneJob (crt/x86_64/crti.s)
???:?:?: 0x7bb13a4 in performAllTheWork (crt/x86_64/crti.s)
???:?:?: 0x7bace7c in update (crt/x86_64/crti.s)
???:?:?: 0x7bd499d in updateModule (crt/x86_64/crti.s)
???:?:?: 0x7ab9d10 in buildOutputType (crt/x86_64/crti.s)
???:?:?: 0x7a933c1 in mainArgs (crt/x86_64/crti.s)
???:?:?: 0x7a92b45 in main (crt/x86_64/crti.s)
???:?:?: 0x7a95227 in main (crt/x86_64/crti.s)
???:?:?: 0x7fa6abd729c9 in ??? (???)
???:?:?: 0x7fffeda2eb52 in ??? (???)
error: uzenatch...
error: The following command terminated unexpectedly:
/home/prokop/source/zig/stage3/bin/zig build-exe /home/prokop/code/uz/src/main.zig --cache-dir /home/prokop/code/uz/zig-cache --global-cache-dir /home/prokop/.cache/zig --name uzenatch --enable-cache 
error: the following build command failed with exit code 6:
/home/prokop/code/uz/zig-cache/o/a6e1935de2c683a7ce518c885c601878/build /home/prokop/source/zig/stage3/bin/zig /home/prokop/code/uz /home/prokop/code/uz/zig-cache /home/prokop/.cache/zig
```

### Expected Behavior

The compiler not to crash

**Comment by schmee** [comment]
Can't reproduce on `0.11.0-dev.1649+07630eb69`, macOS x86.

**Comment by Vexu** [comment]
You need a debug build of the compiler.

**Comment by schmee** [comment]
Sorry about that, I was indeed using the wrong compiler build, with a debug build I do get the same issue.

**Comment by kcbanner** [comment]
I can't reproduce this on a debug or release build of `0.11.0-dev.2757+2d27ad618` (x86_64-windows-gnu). May have been fixed?

**Comment by Vexu** [comment]
Fixed by #14784
